docs/nodes/validate/add-a-validator.md

```diff
@@ -48,182 +48,306 @@ The response has your node’s ID:
 
-![Web wallet earn tab](/img/web-wallet-earn-tab.png)
+Open [the wallet](https://wallet.avax.network/), and go the `Earn` tab. Choose `Add Validator` under the `Validate` section.
 
 Fill out the staking parameters. They are explained in more detail in [this doc](../validate/staking.md). When you’ve filled in all the staking parameters and double-checked them, click `Confirm`. Make sure the staking period is at least 2 weeks, the delegation fee rate is at least 2%, and you’re staking at least 2,000 AVAX.
 
-![Earn validate](/img/earn-validate.png)
+You should a success message, and your balance should be updated.
 
-You should see this success message, and your balance should be updated.
+Calling [`platform.getPendingValidators`](../../apis/avalanchego/apis/p-chain.md#platformgetpendingvalidators) verifies that our transaction was accepted.
 
-![Your validation transaction is sent](/img/your-validation-transaction-is-sent.png)
+Go back to the `Earn` tab, and click `Estimated Rewards`.
 
-Calling [`platform.getPendingValidators`](../../apis/avalanchego/apis/p-chain.md#platformgetpendingvalidators) verifies that our transaction was accepted.
+Once your validator’s start time has passed, you will see the rewards it may earn, as well as its start time, end time, and the percentage of its validation period that has passed.
 
-![getPendingValidators postman](/img/getPendingValidators-postman.png)
+That’s it!
 
-Go back to the `Earn` tab, and click `Estimated Rewards`.
+## Add a validator with AvalancheJS
 
-![Earn, validate, delegate](/img/earn-validate-delegate.png)
+We can also add a node to the validator set using [AvalancheJS](../../apis/avalanchejs/README.md).
 
-Once your validator’s start time has passed, you will see the rewards it may earn, as well as its start time, end time, and the percentage of its validation period that has passed.
+### Install AvalancheJS
 
-![Estimated rewards](/img/estimated-rewards.png)
+To use AvalancheJS, you can clone the repo:
 
-That’s it!
+```zsh
+$ git clone https://github.com/ava-labs/avalanchejs.git
+```
+
+or add it to an existing project:
+
+```zsh
+$ yarn add --dev avalanche
+```
+
+For this tutorial we will use [`ts-node`](https://www.npmjs.com/package/ts-node) to run the example scripts directly from an AvalancheJS directory.
 
-## Add a validator with API calls
+### Fuji Workflow
 
-We can also add a node to the validator set by making API calls to our node. To add a node the Primary Network, we’ll call [`platform.addValidator`](../../apis/avalanchego/apis/p-chain.md#platformaddvalidator).
+Open your AvalancheJS directory and select the [**`examples/platformvm`**](https://github.com/ava-labs/avalanchejs/tree/master/examples/platformvm) folder to view the source code for the examples scripts.
 
-This method’s signature is:
+We will use the [**`buildAddValidatorTx.ts`**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildAddValidatorTx.ts) script to add a validator.
```

Comment:
Also please link to the avalancheJS API `pchain.buildAddValidatorTx` which is the focus point of this section.